docker/pypi/dmwm-base/manage

```diff
@@ -9,17 +9,35 @@
 ##H   start       (re)start the service
 ##H   stop        stop the service
 
-srv=`echo $USER | sed -e "s,_,,g" | sed -e "s,t0req,t0_req,g"`
+# common settings to prettify output
 echo_e=-e
-LOGDIR=/data/srv/logs/$srv
-CFGFILE=/data/srv/current/config/$srv/config.py
-STATEDIR=/data/srv/state/$srv
 COLOR_OK="\\033[0;32m"
 COLOR_WARN="\\033[0;31m"
 COLOR_NORMAL="\\033[0;39m"
+
+# service settings
+srv=`echo $USER | sed -e "s,_,,g" | sed -e "s,t0req,t0_req,g"`
+LOGDIR=/data/srv/logs/$srv
+AUTHDIR=/data/srv/current/auth/$srv
+STATEDIR=/data/srv/state/$srv
+CFGDIR=/data/srv/current/config/$srv
+CFGFILE=$CFGDIR/config.py
+# some MS services uses different config naming convention, therefore we'll
+# adjust CFGFILE assingment
+for c in monitor output ruleCleaner transferor unmerged; do
+    if [ -f $CFGDIR/config-${c}.py ]; then
+        CFGFILE=$CFGDIR/config-${c}.py
+    fi
+done
+
+# necessary env settings for all WM services
 export PYTHONPATH=$PYTHONPATH:/etc/secrets:/data/srv/current/config/$srv
-export X509_USER_CERT=/data/srv/state/$srv/proxy/proxy.cert
-export X509_USER_KEY=/data/srv/state/$srv/proxy/proxy.key
+export X509_USER_KEY=$AUTHDIR/dmwm-service-key.pem
+export X509_USER_CERT=$AUTHDIR/dmwm-service-cert.pem
+export REQMGR_CACHE_DIR=$STATEDIR
+export WMCORE_CACHE_DIR=$STATEDIR
+export WMAGENT_CONFIG=$CFGFILE
+export RUCIO_HOME=$CFGDIR
```

Comment:
For this RUCIO_HOME variable, did you check whether inside this directory there exist an `etc/rucio.cfg` file? Note that it's inside an "etc" directory. AFAIK this is what Rucio client expects to have. You can check this from the workqueue pod.

Reply:
The way how rucio client works is it has default area `/opt/rucio/etc/config.cfg` and we provide on k8s rucio-secret s with this. Therefore, if RUCIO_HOME is not defined it will read from there. Now, I added RUCIO_HOME to align with RPM based deployment/manage script. You should decide how we should handle this. I see few options:
- we remove RUCIO_HOME from manage script and let k8s manifest files to provide and use rucio secrets (all WM services do that now, e.g. https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/reqmgr2ms-monitor.yaml#L127-L129
- we define RUCIO_HOME as it is provided in this PR (and therefore will match RPM deployment procedure) but then we'll need to add either soft link or copy of rucio config from our rucio-secrets `/opt/rucio` area. If this is what we'll follow, the changes will go to `run.sh` instead of manage

Please let me know how you'd like to see this done and I'll make appropriate changes.

Reply:
If Rucio client will fall back to `/opt/rucio/etc/config.cfg`, then I agree with you that we should not do anything with it nor define the RUCIO_HOME environment variable. Thank you for clarifying it.